odoo/addons/base/models/res_lang.py

```diff
@@ -355,6 +373,16 @@ def action_activate_langs(self):
             }
         }
 
+    def name_get(self):
+        """ Add to language name a '(not activated)' when it is not active """
```

Comment:
actually it is the inverse. You add an "activated flag" when lang is active and installed.